src/main/java/org/sagebionetworks/web/client/widget/entity/file/FileTitleBar.java

```diff
@@ -185,19 +195,34 @@ public void onAddToDownloadList() {
 			// TODO: add special popup to report how many items are in the current download list, and link to
 			// download list.
 			FileEntity entity = (FileEntity) entityBundle.getEntity();
-			jsClient.addFileToDownloadList(entity.getDataFileHandleId(), entity.getId(), new AsyncCallback<DownloadList>() {
-				@Override
-				public void onFailure(Throwable caught) {
-					view.showErrorMessage(caught.getMessage());
-				}
-	
-				@Override
-				public void onSuccess(DownloadList result) {
-					jsniUtils.sendAnalyticsEvent(AddToDownloadList.DOWNLOAD_ACTION_EVENT_NAME, AddToDownloadList.FILES_ADDED_TO_DOWNLOAD_LIST_EVENT_NAME, "1");
-					view.showAddedToDownloadListAlert(entity.getName() + EntityBadge.ADDED_TO_DOWNLOAD_LIST);
-					eventBus.fireEvent(new DownloadListUpdatedEvent());
-				}
-			});
+			if (!DisplayUtils.isInTestWebsite(cookies)) {
```

Comment:
Adding a single file to the DL from the file title bar. Use new service if in alpha mode.